tests/test_contractions.py

```diff
@@ -168,3 +168,14 @@ def tests_init():
     assert test._charge == 0
     assert np.allclose(test._coeffs, np.array([1, 2, 3, 4]))
     assert np.allclose(test._exps, np.array([5, 6, 7, 8]))
+
+
+# ToDo: Test norm using actual integrals
+def test_cartesian_gaussian_norm():
+    """Test cartesian_gaussian_norm."""
+    assert np.isclose(
+        [cartesian_gaussian_norm(np.array([0, 0, 0]), 0.25)], [0.2519794355383807303479140]
```

Comment:
Could you cite your sources if you got these numbers from somewhere? If it is a website, a link would be helpful.

Reply:
These values were determined completely arbitrarily and computed by hand.